packages/e2e-tests/test-applications/create-react-app/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+yarn.lock
```

Comment:
we should run some validation on the `.npmrc` and `.gitignore` files we ask folks to set up.

Reply:
👍 https://github.com/getsentry/sentry-javascript/pull/5849